packages/theme-context/src/ThemeContext/createThemeProxy.test.js

```diff
@@ -0,0 +1,112 @@
+import createThemeProxy from "./createThemeProxy";
+
+/* eslint-disable no-console */
+describe("ThemeContext/createThemeProxy", () => {
+  const consoleError = console.error;
```

Comment:
You should be able to use `jest.spyOn` which basically does this mocking for you.